battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java

```diff
@@ -35,25 +36,28 @@
     private Semaphore shipsPlaced = new Semaphore(0);
     private Semaphore waitForSending = new Semaphore(0);
     private Semaphore myTurn = new Semaphore(0);
+    private BorderPane root;
 
-    GameScene(final SocketServer socketServer) throws IOException {
+    GameScene(final SocketServer socketServer, final Output output) throws IOException {
         this.socketServer = socketServer;
         boolean firstPlayer = socketServer.isFirstPlayer();
+        this.output = output;
         if (!firstPlayer) {
             waitForSending.release();
         } else {
             myTurn.release();
         }
     }
 
+
     private Parent createContent() {
-        BorderPane root = new BorderPane();
+        root = new BorderPane();
         root.setPrefSize(DEFAULT_ROOT_WIDTH, DEFAULT_ROOT_HEIGHT);
         Button randomPlacementButton = createRandomButton();
         root.setBottom(randomPlacementButton);
-        enemyBoard = new Board(true);
+        enemyBoard = new Board(true, output);
         enemyBoard.initialize(getMove());
-        playerBoard = new Board(false);
+        playerBoard = new Board(false, output);
```

Comment:
here too(true, false) without looking to Board class I do not know what it means